source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/MediaRenderer.java

```diff
@@ -206,18 +239,109 @@ private ImageView renderPlayButton(
         return playButtonView;
     }
 
-    private FullscreenVideoView renderMediaPlayer(
-            Context context,
-            ViewGroup viewGroup,
-            Media media,
-            HostConfig hostConfig)
+    static class ControllerVisibilityListener implements StyledPlayerView.ControllerVisibilityListener
```

Comment:
why do you use static class?

Reply:
I didn't see you reply to the comment or remove static

Reply:
class has been removed